.gitignore

```diff
@@ -69,3 +69,5 @@ util/Win_Check_Output.txt
 secrets.tar
 id_rsa_*
 /.vs
+.vscode/ipch/b9f57f596f4ca68d/matrix.ipch
```

Comment:
These lines are not necessary, `.vscode/ipch/` is already in .gitignore on master.

Reply:
Yes, but the issue was that it still showed up on my git history, so I added it in to clean up my commits.

Reply:
There are ways to deal with that, actually.

But for now, if you edit the branch on github, it should fix this.

Reply:
Merging in master would also do the job.